R/reliabilityAnalysisFrequentist.R

```diff
@@ -439,7 +452,7 @@ reliabilityFrequentist <- function(jaspResults, dataset, options) {
         allData <- cbind(allData, newData)
         
         # produce footnote for coefficients that are prone to fail with singular matrices, such as lambda6 and omega
-        if (is.nan(model[["cfi"]][["scaleCfi"]][[i]])) 
+        if (any(is.nan(model[["cfi"]][["scaleCfi"]][[i]])))
```

Comment:
if you're using `is.nan` here because `NA` is fine but `NaN` is not, it's fine. Otherwise I'd suggest:
```suggestion
        if (anyNA(model[["cfi"]][["scaleCfi"]][[i]]))
```

Reply:
If I remember correctly, this was specifically for the NaNs

Reply:
then this is fine.